api/cpp/tests/window.cpp

```diff
@@ -28,3 +28,19 @@ TEST_CASE("Basic Window Visibility")
     instance->hide();
     REQUIRE(instance->window().is_visible() == false);
 }
+
+TEST_CASE("Window Scale Factory Existence")
+{
+    using namespace slint::interpreter;
```

Comment:
The window.cpp test is build unconditionally regardless of SLINT_FEATURE_INTERPRETER
So this should probably be fixed in the tests/CMakeLists.txt  (But that's not a new problem)

Reply:
Good point, I'll push a change to master for that.